scrapers/SARJ-LLC.yml

```diff
@@ -41,8 +58,31 @@ xPathScrapers:
         Name: $data//a[@href[contains(.,'/model/')]]/text()
       Tags:
         Name: //div[@class='tag-list']//a/text()
-      Image: //div[@class='img-container']/img/@src
+      Image:
+        selector: //div[@class='img-container']/img/@src
+        postProcess:
+          - replace:
+            - regex: cover
+              with: "wide"
       Studio:
         Name: //meta[@property='og:site_name']/@content
-
-# Last Updated October 9, 2020
+  galleryScraper:
+    gallery:
+      Title: //div[@class='info-container']/h3/a/text()
+      Details: //div[@class='description-text']/p/text()
```

Comment:
in some galleries there are multiple paragraphs eg `https://www.thelifeerotic.com/model/zeo/gallery/20200920
/DRY_LEAVES_1` a concat should be used.

```yaml
        selector: //div[@class='description-text']/p/text()
        concat: "\n\n"
```